{"name": "test_fail_password_empty[--对不起！请输入密码！]", "status": "passed", "attachments": [{"name": "stdout", "source": "d13c5e77-1caa-43c9-acf1-8b02f2481fe4-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "''"}, {"name": "password", "value": "''"}, {"name": "expect", "value": "'对不起！请输入密码！'"}], "start": 1601279849814, "stop": 1601279853738, "uuid": "36b899c7-c210-437b-9fc5-74490e757dbe", "historyId": "19c20d6591c707086577ec0f634f58fd", "testCaseId": "e048db9676071fc597e1087551b47302", "fullName": "TestCases.test_1_loginCase.TestLogin#test_fail_password_empty", "labels": [{"name": "tag", "value": "Test1"}, {"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_1_loginCase"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "HPTSJ003"}, {"name": "thread", "value": "17224-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_1_loginCase"}]}